server/Server.ts

```diff
@@ -123,6 +123,18 @@ export function newServer(config: Config, authProvider: BlaiseIapNodeProvider, b
         res.status(status).json(result);
     });
 
+    // appointment-resource-planning-questionnaires report endpoint
```

Comment:
Would be good to have a test for this in https://github.com/ONSdigital/blaise-management-information-reports/blob/main/server/Server.test.ts, however, it's not been done for any of the other endpoints.